analyses/mwi/dryspells_trigger/09_ecmwf_monthly_skill_dryspells.md

```diff
@@ -258,7 +259,9 @@ end_date="2-1-2020"
 # the mwi_seasonal-monthly-single-levels_v5_interp*.csv contain results when interpolating the forecasts to be more granular
 # but results actually worsen with this
 date_list=pd.date_range(start=f'1-1-{start_year}', end=end_date, freq='MS')
-all_files=[processing.get_stats_filepath(country_iso3,config,date,resolution=resolution,adm_level=1,use_unrounded_area_coords=use_unrounded_area_coords,all_touched=all_touched) for date in date_list]
+all_files=[processing.get_stats_filepath(country_iso3,config,date,resolution=resolution,
```

Comment:
I'm still getting a file not find issue here:
`/Data/private/processed/mwi/ecmwf/seasonal-monthly-individual-members/prate/unrounded-coords/mwi_seasonal-monthly-single-levels_v5_unrounded-coords_2000_1_adm1_stats.csv'`
I looked on the browser version, and there isn't a `unrounded-coords` directory in `prate`

Reply:
I am so sorry! Apparently I set two incompatible vars (`use_unrounded_area_coords=True` and `source_cds=False`). Fixed it now..

Reply:
No worries! I'm anyway blocked by my own issues with insync at the moment :zany_face: